src/main/java/jacchm/footballapp/mapping/dto/StandingDTO.java

```diff
@@ -7,12 +8,28 @@
 @Data
 public class StandingDTO {
 
-    private Integer id;
+    private Integer id = null;
     private String stage;
     private String type;
     private String group;
     private List<TableInputDTO> table;

```

Comment:
powinieneś to zainicjalizować - `private List<TableInputDTO> table = new LinkedList<>();`